_includes/default-header.html

```diff
@@ -60,9 +60,15 @@
           </div>
 
           <div class="navbar-item has-dropdown is-hoverable">
-            <a class="navbar-link" href="{% link code-of-conduct.md %}">Policies</a>
+            <a class="navbar-link">Policies & Procedures</a>
             <div class="navbar-dropdown">
-              <a class="navbar-item" href="{{ site.github.owner_url }}/policies-procedures-and-docs/blob/main/CODE_OF_CONDUCT.md"> Code of Conduct </a>
+              <a class="navbar-item" href="{% link code-of-conduct.md %}"> Code of Conduct </a>
```

Comment:
The`code-of-conduct.md` should then have the metadata on the top

Reply:
It does have metadata ontop and includes the content of the `CODE_OF_CONDUCT.md` file. This is currently displayed as a [page here](https://deploy-preview-764--ols-bebatut.netlify.app/code-of-conduct).

Reply:
I missed the `code-of-conduct.md` file. I saw only `CODE_OF_CONDUCT.md`